web/frontend/react.md

```diff
@@ -0,0 +1,159 @@
+# Building a React App from Scratch
```

Comment:
It might be worth putting a disclaimer at the top of this that it isn't our latest recommendations.

Reply:
lol Awesome! Tear it to shreds!

This is the story for the next step: https://www.pivotaltracker.com/story/show/168410137

It was my understanding that v1 was about getting ducks in a row. I wanted to get through a lot of the markdown slog to prep for whatever the front-end guild decides comes next. Hope it helps!